app/models/user.rb

```diff
@@ -604,6 +632,23 @@ def saw_notification_id(notification_id)
     end
   end
 
+  def bump_last_seen_reviewable!
+    max_reviewable_id = Reviewable
+      .unseen_list_for(self, preload: false, limit: 1)
+      .except(:order)
+      .order(id: :desc)
+      .pluck_first(:id)
+
+    if max_reviewable_id && (!last_seen_reviewable_id || max_reviewable_id > last_seen_reviewable_id)
```

Comment:
```suggestion
    scope = Reviewable.unseen_list_for(self, preload: false)

    if last_seen_reviewable_id
      scope.where('id > ?', last_seen_reviewable_id)
    end

    max_reviewable_id = scope.maximum(:id)

    if max_reviewable_id
```

This can be simplified even further to the above.

Reply:
yay that's a lot nicer! thank you!!